guide/english/certifications/javascript-algorithms-and-data-structures/es6/compare-scopes-of-the-var-and-let-keywords/index.md

```diff
@@ -1,22 +1,20 @@
 ---
 title: Compare Scopes of the var and let Keywords
 ---
-![](//discourse-user-assets.s3.amazonaws.com/original/2X/3/3c8584a085a0deaea66b3400e6321eeadab552a2.jpg)
 
 ![:triangular_flag_on_post:](https://forum.freecodecamp.com/images/emoji/emoji_one/triangular_flag_on_post.png?v=3 ":triangular_flag_on_post:") Remember to use <a>**`Read-Search-Ask`**</a> if you get stuck. Try to pair program ![:busts_in_silhouette:](https://forum.freecodecamp.com/images/emoji/emoji_one/busts_in_silhouette.png?v=3 ":busts_in_silhouette:") and write your own code ![:pencil:](https://forum.freecodecamp.com/images/emoji/emoji_one/pencil.png?v=3 ":pencil:")
 
-### Problem Explanation:
+## Problem Explanation:
 
-We need to change `var` to `let` in our function scope and add `let` to our block scope.
+_Fix the code so that `i` declared in the if statement is a separate variable than `i` declared in the first line of the function._
```

Comment:
Continuing from @Manish-Giri , this is more confusing than ever. What do you mean by separate variable? Running on separate machines? If you really really want to do this, how about:
```suggestion
Change the code so that the variable `i` declared in the if block is separately scoped than the variable `i` declared at the beginning of the function
```

Reply:
Those changes look perfect in my opinion. I didn't mean to confuse anyone, as I said I was trying to avoid giving users the solution in the instructions, I couldn't think of a better way to explain it and instead cited the original exercise instructions: 
![image](https://user-images.githubusercontent.com/12052430/54948646-74a3c500-4f3d-11e9-98ad-3169b7b89c34.png)
If it's confusing maybe we could change those as well.

Reply:
@RandellDawson @moT01 What do you think?

Reply:
yes, sure @thecodingaviator @AdrianSkar, sounds good to me - someone make the commit please and we can give it a final review/merge

Reply:
I tried to commit, but I get the message "You do not have permission to push to this repository. " for some reason.  @AdrianSkar I am fine with @thecodingaviator's suggestion.  Please commit it.  Thanks.